components/picturePage.js

```diff
@@ -0,0 +1,809 @@
+import * as React from "react";
```

Comment:
A best practice is to minimize import statements, so these should be combined. Same with lines 13-22, lines 3-12 + 24 etc.